torch/csrc/jit/mobile/module.h

```diff
@@ -130,12 +130,19 @@ class TORCH_API Module {
     return *cu_.get();
   }
 
+  void set_delete_memory(std::shared_ptr<char> delete_mem) {
```

Comment:
What's the use case of this function? 

Sorry I may not fully follow the offline discussion on this.

Reply:
It's used in `parse_and_initialize_mobile_module`. So we read some memory from the file; but cannot drop that memory yet because it is still used by the Module. So this allows us to attach the memory we got from file and to module so they have the same lifetime.

Reply:
Thanks! Now I remember. It's for the memory part directly from the file, and to make it the same lifetime as the module.